content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -30,7 +28,7 @@ Follow these steps to get started with Istio's ambient mode:
 
 1.  Download the [latest version of Istio](/docs/setup/getting-started/#download) (v1.21.0 or later) with Alpha support for ambient mode.
 
-1.  If you don’t have a Kubernetes cluster, you can deploy one locally using `kind` with the following command:
+1.  For this guide, we will deploy a new local `kind` cluster with the following command:
```

Comment:
```suggestion
1.  Deploy a new `kind` cluster locally with the following command:
```

assumes you have Docker installed but doesn't state it?  (Didn't before)

Reply:
The first step is a `kind` command (after we tell you in the preamble to use `minikube`), so it doesn't really change the assumption from the prev. version.

For this particular quickstart guide, we are having you run `kind` commands, which will require you to install `kind`, which will require you to install `Docker`. I can be more explicit tho.

 If you want to deviate from the "from nothing" local quickstart, just following `content/en/docs/ops/ambient/install/helm-installation/index.md` is probably better.